power_grid_model_c/power_grid_model/include/power_grid_model/common/exception.hpp

```diff
@@ -193,6 +212,11 @@ class DatasetError : public PowerGridError {
     explicit DatasetError(std::string const& msg) { append_msg("Dataset error: " + msg); }
 };
 
+class ExperimentalFeature : public InvalidArguments {
+  public:
+    using InvalidArguments::InvalidArguments;
```

Comment:
```suggestion
```
@nitbharambe i think sonar cloud does not like this. Can you try to fix it? I think this suggestion should do

Reply:
I think this is a false alarm from SonarCloud. We do not need to do anything.